src/operators.jl

```diff
@@ -382,6 +397,11 @@ function _multiply!{T<:Union{GenericAffExpr,GenericQuadExpr}}(ret::Array{T}, lhs
     ret
 end
 
+# this computes lhs.'*rhs and places it in ret
+function _multiplyt!{T<:Union{GenericAffExpr,GenericQuadExpr}}(ret::Array{T}, lhs::SparseMatrixCSC, rhs::Array)
+    _multiply!(ret, transpose(lhs), rhs) # for sparse matrices this should be cheap
```

Comment:
Funny you should mention that, I was literally just this second about to make a post about it.  Part of the reason I didn't actually test how expensive this was is because sparse matrix multiplication is currently broken (i.e. way slower than it should be), so this will all have to change anyway.  See issue #907 .